### ISSUE TITLE
0.9.7-nightly-9c65e17 tsm1 "failed to delete retention policy 'default', failed to created internal storage: retention policy not found: default"

### ISSUE THREAD

**Issue by daviesalex** [influxdb/influxdb#5037]
We saw this after a restart (after crash)

```
[monitor] 2015/12/08 04:40:12 failed to delete retention policy 'default', failed to created internal storage: retention policy not found: default
```

This continued in the logs. The existing data was simply not presented to queries so we deleted it and started everything from scratch.

I notice that another user saw this after a crash in #4121 - also using a very recent nightly.

**Comment by otoolep** [comment]
I too see this, we made some changes very recently around control of the internal stats, looking into this.

**Comment by otoolep** [comment]
@pauldix will provide a blog-post update today, but we made significant changes to tsm1 in the last 24 hours, and you have to remove all pre-existing tsm1 data.

**Comment by dswarbrick** [comment]
I see this error on 0.9.6rc1, but so far not on 0.9.7-nightly-9c65e17.

**Comment by otoolep** [comment]
Thanks for the report. This will be fixed in the 0.9.6 release and the next nightly.
